packages/babel-helper-member-expression-to-functions/src/index.ts

```diff
@@ -432,44 +435,7 @@ const handle = {
     // MEMBER += VALUE   ->   _set(MEMBER, _get(MEMBER) + VALUE)
     // MEMBER ??= VALUE   ->   _get(MEMBER) ?? _set(MEMBER, VALUE)
     if (parentPath.isAssignmentExpression({ left: node })) {
-      if (this.simpleSet) {
```

Comment:
All this section is moved to the `handleAssignment` function below with no changes (other than renaming `this` to `state`).